src/main/java/com/larscheidschmitzhermes/keycloak/events/MonitoringEventListenerProvider.java

```diff
@@ -93,9 +94,12 @@ private synchronized File getOrCreateCounterFile(String fileName) {
 
     private void increaseCounter(File counterFile) {
         try {
-            Long count = Long.parseLong(Files.readAllLines(counterFile.toPath()).get(0));
-            count++;
-            Files.write(counterFile.toPath(), count.toString().getBytes(Charset.forName("UTF-8")));
+            List<String> lines = Files.readAllLines(counterFile.toPath());
+            if (lines != null && lines.size() > 0){
```

Comment:
Hi Martin, in theory, this condition should never be false - `increaseCounter` is never called before the file was created [(see callsite)](https://github.com/hypery2k/keycloak-monitoring-prometheus/blob/master/src/main/java/com/larscheidschmitzhermes/keycloak/events/MonitoringEventListenerProvider.java#L35) So the error situation you are seeing seems to be some kind of illegal state and I have no clue how this can even occur.

I know that the change you propose will fix the errors you see, but also the plugin will just silently fail to write the metric - making the whole metric collection inherently unreliable, with a margin of error we couldn't even quantify. Only thing I can imagine (if we find no _proper_ way to fix the actual faulty condition) would be a config flag "ignore errors" or something like this.

What do you think? Happy to here your proposals!